### PR TITLE
Update Add-AzServiceFabricNodeType command to use location from cluster

### PR DESCRIPTION
**Pull request by mwesigwaguma**
## Mandatory Checklist

- Please choose the target release of Azure PowerShell
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] N/A

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Review by a-santamaria — APPROVED** [review]

File: src/ServiceFabric/ServiceFabric.Test/ScenarioTests/Common.ps1 [hunk 1]
```diff
@@ -47,7 +47,7 @@ function Get-ResourceGroupName
 
 function Get-ClusterName
 {
-    return "azurermsfclustertest";
+    return "azurermsfclustertest2";
```
Comment (vidai-msft) [inline src/ServiceFabric/ServiceFabric.Test/ScenarioTests/Common.ps1:50]: Updating the shared cluster name failed the existing test cases as the name is part of some endpoints. You can use a new cluster name in the test case that needs re-record only.

**Review by vidai-msft — CHANGES_REQUESTED** [review]

**Comment by vidai-msft** [comment]
@mwesigwaguma Please select the target release for this PR in the `Mandatory Checklist` section and update the `ChangeLog.md`.

File: src/ServiceFabric/ServiceFabric.Test/ScenarioTests/Common.ps1 (hunk 1 above)
Comment (mwesigwaguma) [reply src/ServiceFabric/ServiceFabric.Test/ScenarioTests/Common.ps1]: I updated this. Please take a look

**Comment by vidai-msft** [comment]
> @mwesigwaguma Please select the target release for this PR in the `Mandatory Checklist` section and update the `ChangeLog.md`.

@mwesigwaguma Please take a look at the above action items.

**Comment by mwesigwaguma** [comment]
> > @mwesigwaguma Please select the target release for this PR in the `Mandatory Checklist` section and update the `ChangeLog.md`.
> 
> @mwesigwaguma Please take a look at the above action items.

updated

File: src/ServiceFabric/ServiceFabric/ChangeLog.md [hunk 2]
```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 3.3.3
```
Comment (vidai-msft) [inline src/ServiceFabric/ServiceFabric/ChangeLog.md:22]: Need to add the message in the `Upcoming Release` section instead of creating a new version.

**Review by vidai-msft — CHANGES_REQUESTED** [review]

**Comment by vidai-msft** [comment]
@mwesigwaguma You selected `Public Preview` as the target release. Can you please review this doc https://github.com/Azure/azure-powershell/blob/main/CONTRIBUTING.md#target-release-types and confirm if Public Preview is what you desired?

File: src/ServiceFabric/ServiceFabric/ChangeLog.md (hunk 2 above)
Comment (mwesigwaguma) [reply src/ServiceFabric/ServiceFabric/ChangeLog.md]: updated

**Comment by mwesigwaguma** [comment]
> @mwesigwaguma You selected `Public Preview` as the target release. Can you please review this doc https://github.com/Azure/azure-powershell/blob/main/CONTRIBUTING.md#target-release-types and confirm if Public Preview is what you desired?

updated to general release

File: src/ServiceFabric/ServiceFabric/ChangeLog.md [hunk 3]
```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Update location of nodeType to use cluster location in stead of resource group location
```
Comment (YanaXu) [inline src/ServiceFabric/ServiceFabric/ChangeLog.md:21]:
```suggestion
* Updated location of nodeType to use cluster location in stead of resource group location
```

**Review by YanaXu — APPROVED** [review]

**Pull request merged**